scripts/publish_layers.sh

```diff
@@ -42,58 +64,51 @@ rm -rf layer
 
 echo
 echo "Signing layers..."
-./scripts/sign_layers.sh prod $LAYER_ZIP
-
-
-AVAILABLE_REGIONS=$(aws ec2 describe-regions --output json | jq -r '.[] | .[] | .RegionName')
-
-
-# Check region arg
-if [ -z "$1" ]; then
-    echo "Region parameter not specified, running for all available regions."
-    REGIONS=$AVAILABLE_REGIONS
-else
-    echo "Region parameter specified: $1"
-    if [[ ! "$AVAILABLE_REGIONS" == *"$1"* ]]; then
-        echo "Could not find $1 in available regions: $AVAILABLE_REGIONS"
-        echo ""
-        echo "EXITING SCRIPT."
-        exit 1
-    fi
-    REGIONS=($1)
-fi
-
-echo "Starting publishing layers for regions: $REGIONS"
-
+# ./scripts/sign_layers.sh prod $LAYER_ZIP
```

Comment:
I think we still need to sign the layer, don't we?

Reply:
ah, good catch, commented out for testing.